docs/source/conf.py

```diff
@@ -14,7 +14,11 @@
 # They can be extensions coming with Sphinx (named 'sphinx.ext.*')
 # or your custom ones.
 extensions = [
-    'sphinx.ext.autodoc', 'sphinx.ext.githubpages', 'sphinx.ext.napoleon']
+    'sphinx.ext.autodoc',
+    'sphinx.ext.githubpages',
+    'sphinx.ext.napoleon',
+    'sphinx_rtd_theme',
+    ]
```

Comment:
This is not necessary.  I know it's written in RTD theme's doc but it's probably need updated at the moment.

Reply:
reversed the commit

Reply:
> Thanks for the effort, I saw you struggled a little to get this done smile It'd be really nice if you can show some screenshot and check if the quick navigation is present.
> ![2020-10-03T20:31:54](https://user-images.githubusercontent.com/13689192/94992917-83018100-05b7-11eb-96d4-fb82969f498b.png)

yea I think the RTD Theme does not support adding html_sidebars.

I will try to do a fix or just find another more suitable theme

probably gonna find another theme.